src/BloomExe/CollectionTab/LibraryView.cs

```diff
@@ -56,6 +61,19 @@ public partial class LibraryView :  UserControl, IBloomTabArea
 												});
 		}
 
+		private void BackgroundColorsForLinux() {
```

Comment:
I already fixed the others. I missed this one because it doesn't show in the normal collection view. https://github.com/BloomBooks/BloomDesktop/commit/193356e8ad1211370cee0fc9874411fce2362b41